Koha/InvoiceAdjustment.pm

```diff
@@ -0,0 +1,44 @@
+package Koha::InvoiceAdjustment;
```

Comment:
Does feel like `Acquisitions` should be mentioned in the package name somewhere.. either

`Koha::Acquitions::Invoices::Adjusments` or `Koha::Acquitions::InvoiceAdjustments` perhaps.